sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1924,7 +1924,7 @@ Using this information is something we may look at for future revisions of
 ==== sycl_ext_codeplay_kernel_fusion
 
 As the
-link:../experimental/sycl_ext_codeplay_kernel_fusion.asciidoc[sycl_ext_codeplay_kernel_fusion]
+link:../removed/sycl_ext_codeplay_kernel_fusion.asciidoc[sycl_ext_codeplay_kernel_fusion]
 extension also introduces state to a `sycl::queue`, there are restrictions on
 its usage when combined with `sycl_ext_oneapi_graph`. Exceptions with error code
 `invalid` are thrown in the following cases:
```

Comment:
Should this section be rewritten?  It doesn't really make sense to explain the interaction with an extension that doesn't exist.

Reply:
We could probably remove this entire section. What do you think @EwanC?

Reply:
Checked with @EwanC and decided to remove this section. Done in the latest commit.